### PR TITLE
[RFR] Remove GraphicState, unused methods from PostProcessor

### PR DESCRIPTION
**Pull request by tdgunes**
### Contains

As title says, removes GraphicState class and some unused methods from PostProcessor(preparations for its removal too.)

### How to test

Running the game with different configurations must be enough, since its just refactor.

### CODE REVIEW

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

File: engine/src/main/java/org/terasology/rendering/opengl/PostProcessor.java [hunk 1]
```diff
@@ -211,8 +152,8 @@ public void saveScreenshot() {
             return;
         }
 
-        int width = buffers.sceneFinal.width();
-        int height = buffers.sceneFinal.height();
+        int width = buffersManager.getFBO("ocUndistorted").width();
+        int height = buffersManager.getFBO("sceneFinal").height();
```
Comment (emanuele3d) [inline engine/src/main/java/org/terasology/rendering/opengl/PostProcessor.java]: This seems odd. Why take width from one FBO and height from the other?

Comment (emanuele3d) [inline engine/src/main/java/org/terasology/rendering/opengl/PostProcessor.java]: I haven't found any other issue with this PR. If we can address this quickly we might be able to get it merged over the weekend!

Comment (tdgunes) [inline engine/src/main/java/org/terasology/rendering/opengl/PostProcessor.java]: Thanks for finding this one. I've just fixed it. 

_Seems like a copy-paste issue 😭._

**Comment by GooeyHub** [comment]
Hooray Jenkins reported success with all tests good!

**Comment by Cervator** [comment]
Went ahead and merged this as part of prepping another multiplayer build :-)

**Pull request merged**